src/lib/uploader/Web3FunctionUploader.ts

```diff
@@ -9,6 +9,9 @@ import { Web3FunctionSchema } from "../types";
 
 const OPS_USER_API =
   process.env.OPS_USER_API ?? "https://api.gelato.digital/automate/users";
+
+const MAX_SIZE = 10 * 1024 * 1024; // 10 MB
```

Comment:
let's use 2 different limits:
- DOWNLOAD_MAX_SIZE: 1mb
- EXTRACT_MAX_SIZE: 10mb 

as the compressed tarball should be limited to 1mb

Reply:
@goums on the tests that I was fetching cancelled/active tasks I was remembering 4 MB archives, why we're limiting to 1 MB?

Reply:
@goums separated

Reply:
@mkykadir we are limiting archives to 1mb on user api to have DoS vector on on our DB storage